.gitmodules

```diff
@@ -0,0 +1,5 @@
+[submodule "Folly"]
+	path = GitModules/Folly
+	url = https://github.com/facebook/folly.git
+	branch = master
```

Comment:
this needs to be a LKG version, not master

Reply:
The submodules config does not allow specifying a tag or commit.
LKGs are derived from the master branch (if this has changed, this file can be adjusted accordingly).

The actual version "selection" is done by going to the submodule directory and fetching/checking out the desired commit.

Currently, the checked-out version is `v2020.10.05.00`.